composer.json

```
@@ -33,14 +33,14 @@
     ],
     "homepage": "https://www.doctrine-project.org/projects/collections.html",
     "require": {
-        "php": "^7.2 || ^8.0"
+        "php": "^8.1"
     },
     "require-dev": {
         "ext-json": "*",
         "doctrine/coding-standard": "^9.0",
         "phpstan/phpstan": "^1.4.8",
         "phpstan/phpstan-phpunit": "^1.0",
-        "phpunit/phpunit": "^7.5 || ^8.5 || ^9.1.5",
+        "phpunit/phpunit": "^9.1.5",
```

```suggestion
        "phpunit/phpunit": "^9.5",
```

PHPUnit 9.4 and below won't work on PHP 8.1.